src/h3/api/_api_template.py

```diff
@@ -150,8 +150,18 @@ def uncompact(hexes, res):
 
         return _out_collection(hu)
 
-    def polyfill(geos, res):
-        mv = mvi.polyfill(geos, res)
+    def polyfill_polygon(outer, holes=None, res=9, order='latlng'):
+        mv = mvi.polyfill_polygon(outer, holes, res, order)
```

Comment:
Why is this `order` rather than `geojson_conformant`? The latter also implies closed loops.

Reply:
If the C lib doesn't need closed loops, then we don't need to conform to GeoJSON outside of the (lat, lng) vs (lng, lat) order.

It doesn't look like the C lib cares about the right-hand rule either.

Instead of having `geojson_conformant` as an option, I was thinking of having these functions as being as lax as possible, while the `polyfill_geojson()` function would just imply that the input must conform.

Reply:
Also, updated the signature to be `polyfill_polygon(outer, res, holes=None, lnglat_order=False)`